nuxeo-core/nuxeo-core-bulk/src/test/java/org/nuxeo/ecm/core/bulk/action/TestSetPropertyComputation.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     pierre
+ */
+package org.nuxeo.ecm.core.bulk.action;
+
+import static java.util.Collections.singleton;
+import static org.junit.Assert.assertEquals;
+import static org.nuxeo.ecm.core.bulk.action.SetPropertiesAction.ACTION_NAME;
+import static org.nuxeo.ecm.core.bulk.action.SetPropertiesAction.PARAM_DISABLE_AUDIT;
+import static org.nuxeo.ecm.core.bulk.action.SetPropertiesAction.PARAM_VERSIONING_OPTION;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collection;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+import org.nuxeo.ecm.core.api.VersioningOption;
+import org.nuxeo.ecm.core.bulk.BulkServiceImpl;
+import org.nuxeo.ecm.core.bulk.action.SetPropertiesAction.SetPropertyComputation;
+import org.nuxeo.ecm.core.bulk.message.BulkCommand;
+import org.nuxeo.ecm.core.bulk.message.BulkCommand.Builder;
+
+/**
+ * @since 10.3
+ */
+public class TestSetPropertyComputation {
+
+    private static class TestableSetPropertyComputation extends SetPropertyComputation {
+        private TestableSetPropertyComputation(BulkCommand command) {
+            this.command = command;
+        }
+    }
+
+    @Inject
+    public BulkServiceImpl bulkService;
```

Comment:
`BulkService` instead of `BulkServiceImpl`

Reply:
Actually it can even be removed, I'll need your thumb again :-/